index.html

```diff
@@ -52,7 +64,7 @@ <h3>Let's Go For A Walk</h3>
         <p>Suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.</p>
       
         <div class="btn">
-          Sign Up
+            <button type="button">Sign Me Up!</button>
```

Comment:
The wrapping `div` was being used in stead of the more semantic `button` tag. This makes the wrapping `div` obsolete. All it is doing is complicating the styling of the button.